popup/src/stores/network.ts

```diff
@@ -32,6 +32,15 @@ export const useNetwork = defineStore("Network", () => {
 				explorerEndpoint: "https://heimdall.9cscan.com",
 				isMainnet: true,
 			},
+			{
+				id: "0x900000000000",
+				name: "odin",
```

Comment:
Shouldn't this be renamed to `localhost` or something else, instead of `odin`?

Reply:
Okay. I'll change it to `odin-localhost`.

Reply:
Why is the name `odin-localhost`? How is it related to `odin`?

Reply:
NineChronicles nodes are being configured with specific planet names.
- https://github.com/planetarium/NineChronicles.Headless/blob/development/NineChronicles.Headless.Executable/Program.cs#L169
- https://github.com/planetarium/lib9c/blob/development/Lib9c/Planet.cs

The network I've added here is for use as an example in the NineChronicles developer portal.
In this example, the NineChronicles node's planet is set to Odin, so we're including Odin in the name of this network.
- https://github.com/planetarium/www.nine-chronicles.dev/blob/main/en/guide/create-network/running-a-blockchain-node-with-dotnet-project.md?plain=1#L114

Reply:
I okay, I may understand your intention. But I don't agree with the background and the `Planet` type which is for only networks operated by Planetarium. I don't think my opinion is not too important to block your work. You can ignore my comment and merge this pull request.

Reply:
First of all, I'm glad you were able to understand my intentions, and since you are one of the important maintainers of this repository, your opinion is important to me, and that's why I think we should continue this conversation further.

Reply:
> But I don't agree with the background and the Planet type which is for only networks operated by Planetarium.

당신은 크로노가 Planetarium에서 운영하는 네트워크가 아닌 모든 나인크로니클 네트워크를 위해 존재해야 한다고 생각하는 것 같습니다. 그리고 나도 같은 생각입니다.
You seem to think that Chrono should exist for all Nine Chronicles networks, not just the ones run by Planetarium, and I feel the same way.
저는 `odin`이라는 Planet에 정의되어 있는 이름을 내포하는 것이 크로노가 나인크로니클 전체 네트워크를 위하는 것이 아니게 되는 것이라고 이해하지 않습니다. `odin`이나 `heimdall` 그리고 `idun` 등 모든 Planet은 나인크로니클의 일부이기 때문입니다. 그리고 크로노가 이미 기본 네트워크 설정으로 `odin`과 `heimdall`을 포함하고 있는 이유가 여기에 있다고 생각합니다.
I don't understand how including a name defined on a Planet called `odin` means that Chrono is not intended for the entire NineChronicles network. All planets are part of NineChronicles, whether it's `odin` or `heimdall` or `idun`, and I think that's why Chrono already includes `odin` and `heimdall` as default network settings.
`odin-localhost`라고 수정한 것에 대한 당신의 의견을 조금 더 구체적으로 알려주세요.
Please be a little more specific with your comments about the modification to `odin-localhost`.

Reply:
> Please be a little more specific with your comments about the modification to odin-localhost.

I thought there were two Nine Chronicles networks and they were called Odin and Heimdall, respectively. Just like Ethereum has different network names like Mainnet, Sephoria, Gorilla, etc. but you don't necessarily name the local network you open with a new baked Genesis as Mainnet, Sephoria, Gorilla, etc. because it's a different Genesis. It's a different network altogether.

+) Regarding the relationship between odin and odin-internal, odin-internal and odin share the same genesis. It's actually the same network, and if it's PoW, it's the same network where it's normal to mix blocks. So I think it's a slightly different issue than odin-localhost.

Reply:
아하 제네시스 블록 해시를 기준으로 플래닛을 구분하고 있으니, 오딘 플래닛의 제네시스 블록 해시와 다른 해시를 갖고 있는 제네시스 블록을 사용하는 네트워크에 오딘이라는 이름을 사용하는 것이 잘못됐다는 의견이군요. 잘 이해했습니다.
Aha, so you're saying that since we're differentiating between planets based on the genesis block hash, it's wrong to use the name Odin for a network that uses a genesis block that has a different hash than the genesis block hash of the Odin planet. I understand that well.
저는 나인크로니클 노드의 실행 관점에서 생각했는데, 제네시스 블록 관점에서 생각해볼 수 있었습니다. 감사합니다.
I was thinking in terms of the execution of the NineChronicles node, but I could see it in terms of the genesis block. Thanks.
그렇다면 추가하는 네트워크 이름을 `local-network`라고 수정하고, 관련한 코드에 주석을 달아두겠습니다.
In that case, I'll rename the network I'm adding to `local-network` and comment out the relevant code.